src/pages/CoursePage/CoursePage.scss

```diff
@@ -0,0 +1,152 @@
+.course__header {
+  width: 100vw;
+  color: #fff;
+  width: 100vw;
+  background-color: #29303b;
+  transform: translateX(-50%);
+  position: relative;
+  left: 50%;
+}
+
+.course__headerWrapper,
+.course__main {
+  max-width: 1200px;
+  display: flex;
+	justify-content: space-between;
+  margin: 0 auto;
+  padding: 50px 30px;
+}
+
+.course__info {
+  width: 50%;
+  margin-right: 30px;
+}
+
+.course__title {
+  font-size: 34px;
+  margin-bottom: 15px;
+}
+
+.course__description {
+  opacity: 0.8;
+  font-size: 21px;
+  margin-bottom: 20px;
+}
+
+.course__countInfo {
+  display: flex;
+  margin-bottom: 20px;
+}
+
+.course__lessonCount {
+  margin-right: 20px;
+  font-size: 14px;
+}
+
+.course__video {
+	width: 50%;
+  max-width: 500px;
+  max-height: 250px;
+  height: 0;
+  position: relative;
+  top: 0;
+  left: 0;
+  padding-bottom: 28%;
+  transition: filter 0.4s;
+  cursor: pointer;
+  background-color: #fff;
+
+  &:hover {
+    .course__videoPreview {
+      filter: brightness(80%);
+    }
+
+    .course__playButton {
+      background-color: rgba(248, 248, 248, 0.9);
+      transform: translate(-50%, -50%) scale(1.1);
+    }
+  }
+}
+
+.course__playButton {
+  width: 70px;
+  height: 70px;
+  position: absolute;
+  z-index: 1;
+  top: 50%;
+  left: 50%;
+  transition: transform 0.25s;
+  transform: translate(-50%, -50%);
+  background-color: rgba(248, 248, 248, 0.7);
+  border-radius: 100%;
+
+  &::after {
+    content: '';
+    display: block;
+    box-sizing: border-box;
```

Comment:
These styles are already in the file reset.scss  (*,
*::before,
*::after {
  box-sizing: border-box;
})